### DISCUSSION TITLE
ایده ها

### DISCUSSION THREAD

**Discussion by osAIran** [oalee/os-plan-coalition#4] [Ideas]
🙌🏻 درود
🙃 امید وارم حالتون خوب باشه ❤️ هرچه زود تر به عدالت برسیم 🔥 و فردای بهتری با هم بسازیم 🤖. اینجا مکانی برای صحبت و معرفی ایده های شما در باره‌ی ائتلاف و این طرح می‌باشد. این یک برنامه است که ما با هم می‌سازیم 🤗

**Comment by Jiwe-Mobarez** [comment]
دموکراسی گفتمانی نیازمند افرادیست که دنبال این هستن که نتیجه اخری که بدست می اید برای تمامی افراد اون سیستم بهترین باشد. پس باید در طی روند گفتمان به دوستانمان نشان دهیم که سود یک گروه نباید لزوما به ضرر گروه دیگری باشد.

 اگر افراد، هرچقدر که توانمند و با تجربه باشند، خواسته های گروهی خود را به نیاز های جمعی جامعه ترجیح دادند متاسفانه ممکن است در روند دموکراسی اختلال ایجاد شود.

**Comment by osAIran** [comment]
به نظرم ائتلافی که تشکیل میشه باید نماینده‌ی همه ی مردم و اقلیت ها و بهترین مهندس ها و متخصص هامون که سابقه‌ی خوبی دارن هم باشن. باید یک تعریفی از این سابقه ی خوب درست کنیم با هم.

رزومه و این که نماینده‌ی چه قشری هست و اهدافش چیه با لایواستریم و مصاحبه انلاین برگذار بشه و داده آزاد بیاد بعد تحلیل بشه توسط مردم و همه. مردم میتونن مساحبه گیرنده باشن یک livestream ازاد هست که افراد میتونن درخواست join بدن و مثل زوم و google meets برن و هر کسی مثلا ۳ دقیقه حد اقل فرصت داره سوال بپرسه اگه سوالی داره. احترام فقط باید رعایت بشه و این moderation شفاف باشه.
